platform/ui/src/components/InputRange/InputRange.tsx

```diff
@@ -72,14 +86,16 @@ const InputRange: React.FC<{
           inputClassName ? inputClassName : ''
         }`}
         style={{
-          background: `linear-gradient(to right, #5acce6 0%, #5acce6 ${rangeValuePercentage -
-            10}%, #3a3f99 ${rangeValuePercentage + 10}%, #3a3f99 100%)`,
+          background: solidTrackBackground
+            ? '#3a3f99'
```

Comment:
let's make this a configurable option in the props, same for the circle background, it can have defaults but should be able to get configured too

Reply:
Will do for the track, but the circle/thumb is handled in the css file and I am not sure it can be done inline.